lib/internal/crypto/cipherbase.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const LazyTransform = require('internal/streams/lazy_transform');
+const StringDecoder = require('string_decoder').StringDecoder;
+const util = require('util');
+const assert = require('assert');
+const toBuf = require('internal/crypto/util').toBuf;
+
+module.exports = function(binding, crypto) {
```

Comment:
I don't really like having indent for contents of whole file. I wonder if this file could just use circular require instead?